packages/webapp/src/components/Tile/styles.module.scss

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+.typeContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #e8f0eb;
+  box-shadow: 0 2px 10px #ededed;
+  border-radius: 4px;
+  width: 24.4vw;
+  height: 24.4vw;
+  padding: 0 4px;
+  overflow: hidden;
+  cursor: pointer;
+  text-align: center;
+
+  svg {
+    width: 13.3vw;
+    height: 13.3vw;
+    margin-bottom: 4px;
+  }
+
+  font-size: 3.3vw;
+  line-height: 4.4vw;
+  color: var(--fontColor);
+  font-family: 'Open Sans', 'SansSerif', serif;
+}
+
+.typeContainer:hover, .typeContainer:active {
+  border-color: var(--teal700);
+  background-color: var(--green100);
+  color: var(--teal900);
+}
+
+.typeContainerSelected {
+  border-color: var(--teal700);
+  background-color: var(--green100);
+  color: var(--teal900);
+}
+
+.taskTypeLabelContainer {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+}
+
+.matrixContainer {
+  width: 100%;
```

Comment:
⬆️ is the only line added to the copy of `/packages/webapp/src/components/Task/PureTaskTypeSelection/styles.module.scss`